libbindgen/build.rs

```diff
@@ -24,6 +24,7 @@ mod testgen {
         let out_dir = Path::new(env!("OUT_DIR"));
         let mut dst = File::create(Path::new(&out_dir).join("tests.rs")).unwrap();
 
+        println!("cargo:rerun-if-changed=tests/headers");
```

Comment:
I'm not sure this works as you expect according to http://doc.crates.io/build-script.html, but I guess it doesn't hurt.